server/webhook_worker.go

```diff
@@ -80,3 +83,44 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 
 	return nil
 }
+
+func (wh *webhook) FetchReporterNotification(bb []byte, reporter *jira.User) error {
```

Comment:
Can we move this function to a file where we are defining methods on `*webhook`?

```suggestion
func (wh *webhook) applyReporterNotification(bb []byte, reporter *jira.User) error {
```